api/candlepin-api-spec.yaml

```diff
@@ -489,6 +489,60 @@ paths:
         default:
           $ref: '#/components/responses/default'
 
+  /content:
+    get:
+      tags:
+        - Content
+      description: Retrieves list of Content
+      operationId: listContent
+      x-java-response:
+        type: Iterable
+        isContainer: true
+      security: []
+      responses:
+        200:
+          description: Content successfully listed
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/ContentDTO'
+        default:
+          $ref: '#/components/responses/default'
+
+  /content/content_uuid:
```

Comment:
Minor issue - Use `{}` here for path parameter.

Reply:
Good catch. Removed the spec test and fixed those braces.